spec/models/user_spec.rb

```diff
@@ -15,4 +15,39 @@
     assert user.reload.access_locked?
   end
 
+  describe '.send_reset_password_instructions' do
+
+    context 'when the user hasn\'t requested a reset password token before' do
```

Comment:
You should test what happens when an email is password for an unknown user and then verify that we  continue to get the same behavior as we did before.

Reply:
Done